deepwalk/__main__.py

```diff
@@ -150,6 +157,9 @@ def main():
   parser.add_argument('--workers', default=1, type=int,
                       help='Number of parallel processes.')
 
+  parser.add_argument('--pretrained', nargs='?',
+                      help='Pre-trained embeddings file')
```

Comment:
What should be the format of the embedding file?

Reply:
Yes, I agree. The format is the same as the output of the model, which gensim's docs refer to as C's  Word2Vec format. I don't know exactly how to refer to that format in the help message.